docs/user-guide/09-targets.md

```diff
@@ -167,14 +163,15 @@ Revisions to D3D12 have added additional stages to the rasterization pipeline, a
 
 #### Mesh Shaders
 
-> #### Note ###
+> #### Note
+>
```

Comment:
We do support mesh and amplification shaders for metal.

Reply:
Oh, this is in the dx12 section (and I guess was an old bit of documentation?) My editor must have formatted the file and created a diff touching in more places.